net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -28,8 +28,8 @@ DEPEND="${RDEPEND}
 	i2p-hardening? ( >=sys-devel/gcc-4.7 )
 	|| ( >=sys-devel/gcc-4.7 >=sys-devel/clang-3.3 )"
 
-I2PD_USER="${I2PD_USER:-i2pd}"
-I2PD_GROUP="${I2PD_GROUP:-i2pd}"
+I2PD_USER=i2pd
+I2PD_GROUP=i2pd
 
```

Comment:
IMO taking this out takes away customizability from the user. Taking it out here would mean to take it away from the user. But then again, we don't enforce these vars in our config files. Enforcing them makes it less foolproof but more static.
Cleaning up, it's fine.